src/twisted/trial/_asynctest.py

```diff
@@ -28,6 +29,18 @@
 _wait_is_running: List[None] = []
 
 
+async def _maybeCoroutine(*args, **kw):
```

Comment:
this _maybeCoroutine is only used once and adds an extra frame, it's better to inline it